conda/exceptions.py

```diff
@@ -1297,16 +1297,31 @@ def __init__(self, msg: str, *args, **kwargs):
 
 
 class EnvironmentSpecPluginNotDetected(SpecNotFound):
-    def __init__(self, name: str, plugin_names: Iterable[str], *args, **kwargs):
+    def __init__(
+        self,
+        name: str,
+        plugin_names: Iterable[str],
+        autodetect_disabled_plugins: Iterable[str] = [],
```

Comment:
```suggestion
        autodetect_disabled_plugins: Iterable[str] = (),
```

Immutable is better.